implementations/rust/ockam/ockam_ffi/Cargo.toml

```diff
@@ -15,9 +15,6 @@ description = """FFI layer for ockam_vault.
 [lib]
 crate-type = ["staticlib", "rlib", "cdylib"]
 
```

Comment:
It would be great if Cargo had a way to select crate-type per target so we can get rid of:

```
warning: dropping unsupported crate type `cdylib` for target `thumbv7em-none-eabihf`

warning: `ockam_vault` (lib) generated 1 warning
```

Reply:
In `.cargo/config.toml` you can put
```
[target.<triple>]
rustflags = ["--crate-type", "staticlib"]
```
But I don't know if cargo will still pass it's --crate-type flags in addition.